_includes/venue.html

```diff
@@ -2,10 +2,25 @@
     <div class="container">
         <div class="row">
             <div class="col-sm-8 col-sm-offset-2 text-center">
-                <h2 class="section-heading">Back on Site!</h2>
-                <hr class="primary">
-                <p>Venue to be announced soon. Stay tuned!</p>
+                <h2 class="section-heading">Venue</h2>
+                <hr class="light">
+                <p>We are beyond elated to be <strong>back on site</strong> after two virtual editions.</p>
+                <p>Thanks a lot to <a target="_blank" rel="noopener" href="https://www.scaleway.com/">Scaleway</a> to
```

Comment:
change to "for accommodating us"

Reply:
fixed in ccef5e155fc418b3a2560ea866b26fc5852a6990